articles/ros2_sectest.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: default
+title: ROS 2 Security Testing
+permalink: articles/ros2_sectest.html
+abstract:
+  This document describes a proposed package for testing
+  security concerns specific to ROS 2.
+author: >
+  [Thomas Moulard](https://github.com/thomas-moulard),
+  [Ryan Newell](https://github.com/ryanewel)
+
+published: true
+categories: Security
+---
+
+{:toc}
+
+{::options parse_block_html="true" /}
+<style>
+table {
+    table-layout: fixed;
+    width: 100%;
+}
+
+th {
+    width: 11em;
+}
+</style>
+
+# {{ page.title }}
+
+## Intent Of This Document
+The intent of this document is to:
+
+1. Describe the design of a new ROS 2 packages called `ROS2-SecTest`. This package will gather code which can be used to "attack" ROS 2 applications.
```

Comment:
Since `ROS2-SecTest` isn't a valid package name please use something like `security_testing` instead. Also "ROS2" in the package name doesn't make much sense since every package is a ROS (2) package.

Reply:
Good point, changed the package name